uv.schema.json

```diff
@@ -1397,6 +1461,12 @@
         }
       ]
     },
+    "Sources": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Source"
+      }
+    },
```

Comment:
I think this is wrong (it thinks it _has_ to be a list).

Reply:
(I think I fixed it between the time I wrote this comment and your testing. Thanks so much for testing it.)